.github/workflows/stable-release.yml

```diff
@@ -42,7 +42,7 @@ jobs:
             false
           fi
 
-          echo "::set-output name=DRAFT_RELEASE_RUN_ID::${DRAFT_RELEASE_RUN_ID}"
+          echo "{DRAFT_RELEASE_RUN_ID}=${DRAFT_RELEASE_RUN_ID}" >> $GITHUB_OUTPUT
```

Comment:
We have a gh_output function in our shell script library that could be used here.

Reply:
Thank you, I switched to it in 517978d.

Reply:
Both places where `set-output` is used is done like that because we do not checkout the `syslog-ng` git repo. Both jobs are acquiring `syslog-ng` from other sources, like the `stable-release` job through the tarball and the `test-apt-packages` job through `apt`. We intentionally refused to checkout the git repo to make sure that no source code is necessary for those tasks.

I think these should be kept this way, but even if we are okay with sourcing the script, we should add a checkout first.

Also, please test the modified jobs in your fork, as some of them are not executed under PRs. We could have broken `stable-release` and `test-apt-packages` if we were not careful.

Reply:
I didn't know this, sorry. Makes sense, but maybe a comment would be great to have in those workflows so this information is recorded.

Reply:
Thanks for the heads up @alltilla, will do test runs and then get back to it :+1:

I could also add a comment as well about the intention, let me know if I should do so.

Reply:
Thank you! Yes, if you could do that I would be really happy :)

Reply:
I tried running the modified jobs in my fork, but [create-packages / debian-stretch](https://github.com/OverOrion/syslog-ng/actions/runs/3725510797/jobs/6318380310#logs) failed. I checked out master and currently it fails as well, which is why I submitted a new issues for that: #4246.

Reply:
This one got resolved by me dropping Debian stretch support.